Cython/Compiler/Main.py

```diff
@@ -9,15 +9,6 @@
 import sys
 import io
 
-if sys.version_info[:2] < (2, 7) or (3, 0) <= sys.version_info[:2] < (3, 3):
```

Comment:
This line maybe should not be just deleted but adjusted to lowest supported version of python.

Reply:
Yes, I think so

Reply:
Fixed.